src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,41 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+    @Test
+    void Split(){
+        String[] result = "1,2".split(",");
+        assertThat(result).containsExactly("1", "2");
+    }
+
+    @Test
+    void SubString(){
+        String result = "(1,2)".substring(1,4);
+        assertThat(result).isEqualTo("1,2");
+    }
+
+    @Test
+    @DisplayName("result의 인덱스 범위를 벗어나면 IndexOutOfBoundsException발생하는지 예외에 나타나는 숫자까지 테스트")
+    void 인덱스_범위_테스트_예외메시지_숫자포함(){
+        String result = "abc";
+        assertThatThrownBy(() -> {
+            result.charAt(3);
```

Comment:
```suggestion
        assertThatThrownBy(() -> {
            "abc".charAt(3);
```
굳이 불필요한 로컬 변수는 사용하지 않아도 되지 않을까?